### ISSUE TITLE
Upgrade XNAT

### ISSUE THREAD

**Issue by pipitone** [TIGRLab/admin#254]
We're currently running a xnat server on `xen.kimel.ric`, but we need to upgrade this to the latest version of XNAT (@josephdviviano and I decided to wait until XNAT 1.6.5 since 1.6.4 has jar dependency problems). 

Completing this should consist of minimally: 
- [x] Using `kimel-web-01` for a new XNAT install 
- [x] Importing the current XNAT database and upgrading
- [x] Updating the `kimel-web-01` ansible script, as well as putting any other configuration (cron jobs, etc..) under ansible control

**Comment by pipitone** [comment]
XNAT 1.6.5 released: https://wiki.xnat.org/display/WaterCooler/2015/08/28/XNAT+1.6.5+is+now+available

**Comment by pipitone** [comment]
A progress report. I've been able to set up XNAT on this server using the XNAT [Vagrant provision.sh script](https://bitbucket.org/nrg/xnat_vagrant_1_6dev) as a guide. It runs flawlessly on Ubuntu 14.04. 

I've also successfully imported and upgrade the current XNAT database by dropping the new database, importing a snapshot of our current database, and running the XNAT upgrade scripts. Roughly: 

```sh 
sudo service tomcat7 stop
sudo su - xnat
dropdb xnat
pg_restore -d xnat ~localadmin/2015-10-13.sqlc
cd /data/xnat/src/xnat-1.6.5
bin/update.sh -Ddeploy=true
psql -f deployments/xnat/sql/xnat-update.sql  xnat
sudo service tomcat7 start
```

My next step will be to use AUFS/OverlayFS to make a writeable fork of our existing XNAT archive and test out XNAT upload/project creation/etc.. functions.
